styles/Canonical/001-English-words-spelling-suggestions.yml

```diff
@@ -59,7 +58,6 @@ swap:
   'kilometer': kilometre
   'labeled': labelled
   'labor': labour
-  'license': licence
```

Comment:
noun/verb

Reply:
Sorry I missed this the first round. The noun / verb usage needs context to validate, unfortunately.